prow-configurations/README.md

```diff
@@ -13,9 +13,19 @@ Install the following tools:
 - [gcloud](https://cloud.google.com/sdk/gcloud/)
 - OpenSSL
 
+### Provision cluster
+Follow k8s/prow documentation steps to provision cluster on GKE (https://github.com/kubernetes/test-infra/blob/master/prow/getting_started.md#create-the-cluster).
+
 ## Installation
+Ensure that kubectl is pointing to correct cluster. For GKE execute following command:
+```
+gcloud container clusters get-credentials {CLUSTER_NAME} --zone={ZONE_NAME} --project={PROJECT_NAME}
+```
 
-Set an OAuth2 token that has read and write access to the bot account. You can set it either as an environment variable named `OAUTH` or interactively during the installation.
+Set an OAuth2 token that has read and write access to the bot account. You can set it either as an environment variable named `OAUTH` or interactively during the installation. 
+To generate new token, on Github page select given account --> Settings --> Developer Settings --> Personal Access Token --> Generate New Token. 
```

Comment:
``To generate a new token, go to the **Settings** tab of a given GitHub account and click **Developer Settings**. Choose **Personal Access Token** and **Generate New Token**.``